test/compound/setup/TestSetup.sol

```diff
@@ -15,7 +15,7 @@ import "../helpers/SimplePriceOracle.sol";
 import "../helpers/DumbOracle.sol";
 import {User} from "../helpers/User.sol";
 import {Utils} from "./Utils.sol";
-import "@config/Config.sol";
+import "config/compound/Config.sol";
```

Comment:
Just a note: adding `config/=config/` to `remappings.txt` would enable the VSCode Solidity extension to find files from import paths with right-click

So I'd suggest to add:
```txt
src/=src/
test/=test/
config/=config/
```

to `remappings.txt`

Reply:
Ok I didn't know that, I going to add that